tests/e2e-pw/utils/navigation/navigation.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import { PlaywrightTestArgs } from '@playwright/test';
+import { BlockData } from '@woocommerce/e2e-types';
+
+/**
+ * Closes any modals in the editor if they are open.
+ */
+export const closeModalIfExists = async (
+	page: PlaywrightTestArgs[ 'page' ]
+) => {
+	// The modal close button can have different aria-labels, depending on the version of Gutenberg/WP.
+	// Newer versions (WP >=6.2) use `Close`, while older versions (WP <6.1) use `Close dialog`.
+	const closeButton = await page.$(
```

Comment:
It always better to use `[locator](https://github.com/woocommerce/woocommerce-blocks/blob/fa3f9856629dcbf77d12fc9064b34b9820574776/docs/contributors/e2e-guidelines.md/#L45-L46)`